app/models/publishing_user.rb

```diff
@@ -0,0 +1,15 @@
+class PublishingUser < ApplicationRecord
```

Comment:
since there's a many-to-many relationship with organisations, there needs to be a `has_and_belongs_to_many :organisations`

I'll make sure to add the same relations on the org model too